app/javascript/mastodon/features/standalone/compose/index.js

```diff
@@ -9,7 +9,7 @@ export default class Compose extends React.PureComponent {
   render () {
     return (
       <div>
-        <ComposeFormContainer />
+        <ComposeFormContainer autoFocus />
```

Comment:
Added autoFocus based on the following.

https://github.com/mastodon/mastodon/pull/16517/files#diff-d1070f16b64790b2deba9983d6aabc28be3b98d8c2f0b55929346a97db53bc92R119